### DISCUSSION TITLE
Winget package for IrfanView plugin?

### DISCUSSION THREAD

**Discussion by cheesecracker** [microsoft/winget-pkgs#23925] [General]
Hello everyone,

I found the winget command to install IrfanView at https://winget.run/pkg/IrfanSkiljan/IrfanView.

But I don't seem to find the plugin which is also important in some situations.

Thanks

**Comment by ImJoakim** [comment]
Hey there @cheesecracker, based on your post, I'm assuming that you want someone to add a separate program/package. If so, can you please create this request under https://github.com/microsoft/winget-pkgs/issues?

The direct link for a package request/submission can be found here: [Issue: Package Request/Submission](https://github.com/microsoft/winget-pkgs/issues/new?assignees=&labels=Help-Wanted%2C+Package-Request&template=Manifest_Request.md&title=)

**Reply by cheesecracker** [reply]
Thank you.
I have just submitted the request.  It's at
https://github.com/microsoft/winget-pkgs/issues/23933

**Comment by ItzLevvie** [comment]


**Reply by cheesecracker** [reply]
Oh I see.  So the package actually exists.
What is the command?  Like this?
winget install -e --id IrfanSkiljan.IrfanView-plugins

**Reply by cheesecracker** [reply]
Thanks for your reply.
After I go through those robot messages and human messages, I understood that the package never existed.
So I made the following comment but I'm not sure if anybody is reading it:

> Can we create a new package to include both the IrfanView AND the plugin?
> We can call the package IrfanSkiljan.IrfanView-all-in-one, to avoid messing up with the original package.
